cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -10,9 +10,11 @@
 import sys
 from dataclasses import dataclass
 from distutils.version import LooseVersion
+from typing import Any, Dict, List, Tuple
```

Comment:
it the main readme, should we put the diagrams from the slide deck?

Reply:
https://flyte--490.org.readthedocs.build/projects/cookbook/en/490/auto/case_studies/ml_training/spark_horovod/index.html -- added two slides.